### ISSUE TITLE
could we wrap the ping payload with a bracket ?

### ISSUE THREAD

**Issue by RonnieRen** [blackhorse-one/stomp_dart_client#30]
for ping payload, we just send a newline  ("\n") as its payload, however for common message sent, the payload is wrapped with a bracket like this: 
```
["SEND\ndestination:/app/webMessage\ncontent-length:236\n\n{"\"content\":\"Greeting from client (2020-08-25 15:45:26.904648)\",\"senderType\":0,\"type\":\"text\",\"flag\":\"1598341526907\",\"createTime\":\"2020-08-25 15:45:26.907645\"}\u0000"]
```
could we keep them be consistent?  currently if my client sent a newline as a ping payload, my backend would immediately close my connection with code=1007.

if I wrap the newline with bracket like this: '[\n]' for ping payload, it works.

**Comment by KammererTob** [comment]
Hey.

This is for SockJS, right? the Stomp protocol should not add any brackets to the payload. This seems like a array represented in JSON. If it is SockJS i would need to check how these type of messages are normally handeled

**Comment by RonnieRen** [comment]
yes, it is for sockJS

**Comment by KammererTob** [comment]
Hey.

i pushed a fix for that. Can you please point your pubspec directly to the reprository and see if this fixes it for you? If so i will release a new version! Thanks for your help!

**Comment by RonnieRen** [comment]
yeah, it fix my problem. thanks.

**Comment by KammererTob** [comment]
Released with 0.3.7
